NextcloudTalk/NCChatViewController.m

```diff
@@ -2722,6 +2722,10 @@ - (void)updateRoomInformation
 
 - (BOOL)shouldPresentLobbyView
 {
+    if ([[NCDatabaseManager sharedInstance] serverHasTalkCapability:kCapabilityChatPermission] && (_room.permissions & NCPermissionCanIgnoreLobby) != 0) {
```

Comment:
We should check for `conversation-permissions` capabilty instead of `chat-permission` capabilty.
Because that's when `Can ignore lobby` premission was introduced.

Since `conversation-permissions` capability was not added in Talk 13 release, you can check it using this workaround:
https://github.com/nextcloud/talk-ios/blob/master/NextcloudTalk/CallViewController.m#L260-L264